packpack

```diff
@@ -136,6 +135,7 @@ else
         echo "usermod -a -G sudo $USER 2>/dev/null || :";
         echo "usermod -a -G wheel $USER 2>/dev/null || :";
         echo "usermod -a -G adm $USER 2>/dev/null || :";
+        printf 'printf "%s\\tALL=(ALL)\\tNOPASSWD: ALL\\n" >> /etc/sudoers\n' $USER
```

Comment:
Comes from a119cafc9d64ad346b9ac1face1625564414c34d? It would be good to rebase on top of the recent master.

Reply:
done